packages/app/src/components/Settings/DomainEditModal.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+const DomainEditModal = ({
+    show,
+    handleClose,
+    domain,
+    handleChange,
+    onSubmit,
+}) => {
+    const showHideClassName = show
+        ? 'edit-modal edit-modal-display-block'
+        : 'edit-modal edit-modal-display-none';
+
+    return (
+        <div className={showHideClassName}>
+            <div className="edit-modal-main">
+                {domain ? (
```

Comment:
Instead of doing a ternary like `{condition ? <MyComponent/> : null}`, you can use conditional rendering like this `{condition && <MyComponent/>}`. If `condition` is false, then `<MyComponent/>` won't render at all.

Reply:
It's been updated.